src/components/DatePicker/_DatePicker.scss

```diff
@@ -1,42 +1,101 @@
 // Styles needed to override `react-datepicker` plugin styles.
 .date-picker-calendar {
+  font-family: var(--nypl-fonts-body);
+
+  .react-datepicker {
+    border-color: var(--nypl-colors-ui-gray-medium);
+  }
+  // The styles for the triangle arrow above the calendar popup.
+  .react-datepicker__triangle {
+    // Shift the triangle to the left.
+    transform: translate(32px, 0px) !important;
+    &::before {
+      border-bottom-color: var(--nypl-colors-ui-gray-medium) !important;
+    }
+    &::after {
+      border-bottom-color: var(--nypl-colors-ui-gray-x-light-cool) !important;
+    }
+  }
+  // The left and right arrow buttons.
+  .react-datepicker__navigation {
+    &-icon {
+      top: 5px;
+      width: 0px;
+      &::before {
+        border-color: var(--nypl-colors-ui-gray-medium);
+      }
+    }
+    &:hover {
+      .react-datepicker__navigation-icon::before {
+        border-color: var(--nypl-colors-ui-gray-dark);
+      }
+    }
+  }
+  // The header with the appropriate date.
+  .react-datepicker__header {
+    border-color: var(--nypl-colors-ui-gray-medium);
+    background-color: var(--nypl-colors-ui-gray-x-light-cool);
+  }
   // Make sure the width is enough to fit 12 selectable years.
   .react-datepicker__year-wrapper {
     max-width: 220px;
   }
-
+  // Hover state
+  .react-datepicker__year-text:not(.react-datepicker__year-text--disabled),
+  .react-datepicker__month-text,
+  .react-datepicker__day {
+    &:hover {
+      border-radius: 0;
+      background-color: var(--nypl-colors-ui-gray-x-light-cool);
+      color: var(--nypl-colors-ui-black);
+    }
+  }
+  .react-datepicker__day--keyboard-selected {
+    opacity: 1;
+    border-radius: 0;
+    background-color: var(--nypl-colors-ui-gray-x-light-cool);
+    color: var(--nypl-colors-ui-black);
+  }
   // Moving away from React DatePickers default blue color theme:
   .react-datepicker__year,
   .react-datepicker__month,
   .react-datepicker__day {
     // For currently selected ranges
     &--in-range.react-datepicker__day--in-range {
-      opacity: 0.5;
+      opacity: 1;
     }
-
     // For ranges about to be selected.
     &-text--keyboard-selected,
     &--keyboard-selected,
     &--in-range,
     &--in-selecting-range {
-      background-color: var(--nypl-colors-ui-black);
+      background-color: var(--nypl-colors-ui-gray-light-cool);
```

Comment:
Becuase I am looking at what’s happening with the code on line 75, I can now see that this should be set to `var(--nypl-colors-ui-gray-x-light-cool)`